### PR TITLE
Core added

### PR DESCRIPTION
**Pull request by PaterSantyago**

### CODE REVIEW

File: src/Lykke.Quintessence.Core.Abstractions/Blockchain/Strategies/IBuildRawTransactionStrategy.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace Lykke.Quintessence.Core.Blockchain.Strategies
+{
+    public interface IBuildRawTransactionStrategy
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Core.Abstractions/Blockchain/Strategies/IBuildRawTransactionStrategy.cs:3]: Not necessary to fix but:
A way to detailed interface, Icon does not use such interface

File: src/Lykke.Quintessence.Core.Abstractions/Blockchain/IWalletGenerator.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Lykke.Quintessence.Core.Blockchain
+{
+    public interface IWalletGenerator
+    {
+        Task<(string Address, string PrivateKey)> GenerateWalletAsync();
```
Comment (cassade) [inline src/Lykke.Quintessence.Core.Abstractions/Blockchain/IWalletGenerator.cs]: Maybe, it's better to implement full contract and add `AddressContext` property to result.

File: src/Lykke.Quintessence.Core/Crypto/HashCalculator.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+namespace Lykke.Quintessence.Core.Crypto
+{
+    [UsedImplicitly]
+    public class HashCalculator : IHashCalculator
+    {
+        // ReSharper disable once InconsistentNaming
+        public static HashCalculator Blake2b256
+            => new HashCalculator(HashType.BLAKE2B_256);
+        
+        public static HashCalculator Keccak256
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Core/Crypto/HashCalculator.cs:15]: Add Sha3 here as well, please

File: src/Lykke.Quintessence.Core/Lykke.Quintessence.Core.csproj [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+      <ProjectReference Include="..\Lykke.Quintessence.Core.Abstractions\Lykke.Quintessence.Core.Abstractions.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Autofac" Version="4.8.1" />
+      <PackageReference Include="Multiformats.Hash" Version="1.4.0" />
+      <PackageReference Include="Nethereum.Signer" Version="3.0.0" />
+      <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
+    </ItemGroup>
+
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Core/Lykke.Quintessence.Core.csproj:26]: Make sure you ship pdb with nuget:
image

**Review by OttoVT — CHANGES_REQUESTED** [review]

File: src/Lykke.Quintessence.Core.Abstractions/Blockchain/Strategies/IBuildRawTransactionStrategy.cs (hunk 1 above)
Comment (PaterSantyago) [reply src/Lykke.Quintessence.Core.Abstractions/Blockchain/Strategies/IBuildRawTransactionStrategy.cs:3]: Open an Issue, please. Will fix it in next release (for BIL 2.0)

**Review by OttoVT — APPROVED** [review]

**Pull request merged**